src/frontend/shortlist/src/components/preferences/__tests__/PreferenceMultiSelect.test.js

```
@@ -0,0 +1,51 @@
+/*
+TODO(All these vue files do not have props that correspond to the API, needs to be added)
+The following tests conclude all major component testing.
+*/
+import { describe, it, expect, vi } from "vitest";
+import { mount } from "@vue/test-utils";
+import PreferenceMultiSelect from "../PreferenceMultiSelect.vue";
+import { createTestingPinia } from "@pinia/testing";
+
+describe("PreferenceMultiSelect.vue", () => {
+  // Check if component exists
+  it("renders with null props", () => {
+    const componentWrapper = mount(PreferenceMultiSelect, {
+      props: {
+        /*
+      TODO(When API call is clear, we will change this to match API variables)
+     */
+        question: "String",
```

out of curiosity, is this setting the value of question to be "String" and the value of the field to be "String" as well? or this an attempt at setting the expected type? 

I ask because for something like multiselect, we actually pass in a list for model value, not a pure string.  The test as written will work because it's only checking the outer component, but the inner components are silently failing here (I think, unless I got super detailed with input handling when I wrote them originally).  It's possible to write a test with bad inputs and then set their conditions to expect them to fail (thereby making the test actually passing) but in this instance if we're just going to test the outer component, let's make sure to change the modelValue prop to be a correctly formed input to reflect that this test isn't really covering us in instances where bad values come in

---

I had some trials and errors and settled with this solution. I know it is not the optimal one. I wanted to find the matching API that will tell me what is a string, what is a list, and what is dictionary etc, but can't find it anywhere in this repo so far? Let me know where? I want to make sure the prop to correctly formed input to reflect in this test